scene/3d/sprite_3d.cpp

```diff
@@ -486,7 +486,7 @@ void Sprite3D::_draw() {
 		RS::get_singleton()->immediate_normal(immediate, normal);
 		RS::get_singleton()->immediate_tangent(immediate, tangent);
 		RS::get_singleton()->immediate_color(immediate, color);
-		RS::get_singleton()->immediate_uv(immediate, uvs[i]);
+		RS::get_singleton()->immediate_uv(immediate, uvs[index[i]]);
```

Comment:
Ouch, how did that not trigger crashes until now? (Or maybe it did :D)

Reply:
Sprite3D do not render anything from me, should be another bug, but it's possible this code is never called.